### PR TITLE
Question section  faq page

### PR DESCRIPTION
**Pull request by abdulbasit-web**
finish accordian drop down question section

### CODE REVIEW

File: src/components/Questions/Questions.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+* {
+  background: #f9f9fa;
+  margin: 0;
+}
+
+.qustion__card {
+  border-radius: 12px;
+  box-shadow: 0px 2px 6px rgba(0, 0, 0, 0.25);
+}
+.card-header {
```
Comment (MustafaNawzat) [inline src/components/Questions/Questions.css:10]: leave one empty line between blocks. here and all

File: src/components/Questions/Questions.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.card-header p {
+  margin: 0;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 24px;
+  line-height: 28px;
+  /* identical to box height */
```
Comment (MustafaNawzat) [inline src/components/Questions/Questions.css:25]: remove comments before pushing. here and all

File: src/components/Questions/Questions.css (hunk 1 above)
Comment (abdulbasit-web) [reply src/components/Questions/Questions.css:11]: fixed

File: src/components/Questions/Questions.css (hunk 2 above)
Comment (abdulbasit-web) [reply src/components/Questions/Questions.css]: fixed

**Pull request closed without merging**